addon/models/preprint.js

```diff
@@ -27,10 +28,32 @@ export default OsfModel.extend({
     isPublished: DS.attr('boolean'),
     isPreprintOrphan: DS.attr('boolean'),
     licenseRecord: DS.attr(),
+    reviewsState: DS.attr('string'),
+    dateLastTransitioned: DS.attr('date'),
 
     // Relationships
     node: DS.belongsTo('node', { inverse: null, async: true }),
     license: DS.belongsTo('license', { inverse: null }),
     primaryFile: DS.belongsTo('file', { inverse: null }),
     provider: DS.belongsTo('preprint-provider', { inverse: 'preprints', async: true }),
+    actions: DS.hasMany('action', { inverse: 'target', async: true }),
+    contributors: DS.hasMany('contributors', { async: true }),
+
+    uniqueSubjects: Ember.computed('subjects', function() {
+        if (!this.get('subjects')) return [];
+        return this.get('subjects').reduce((acc, val) => acc.concat(val), []).uniqBy('id');
+    }),
+
+    doiUrl: Ember.computed('doi', function() {
+        return `https://dx.doi.org/${this.get('doi')}`;
+    }),
```

Comment:
This is probably not necessary as the doi url is in preprint.links.
Depending on which DOI you need:
`preprint.links.doi`  - peer-reviewed article DOI
`preprint.links.preprint_doi` - DOI for the preprint itself